airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -957,19 +954,35 @@ def list_by_timespan(
                 break
         return ids
 
-    def get_size(self, bucket_name: str, object_name: str) -> int:
+    def get_blob(self, bucket_name: str, object_name: str) -> Blob:
```

Comment:
I think that it would be better to make this method internal (`_get_blob`), as it's only used by the other methods here.

Reply:
Great feedbacks, will make sure to apply them. Thanks